app/views/idv/shared/_mini_logo.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= image_tag(
+      asset_url('logo.png'),
```

Comment:
I think vector `.svg` would be preferable in the application where we can use it.

Reply:
Related thread: https://gsa-tts.slack.com/archives/C04GA9WQ85B/p1679078822473909

As discussed there, the interpolated string would make it difficult to search for all instances of `logo.svg`.

Alternatively, this is a very small partial, and maybe it's not worth DRY-ing at all, vs. duplicating the markup.